src/ddprof_worker.cc

```diff
@@ -390,10 +397,12 @@ DDRes ddprof_worker_init(DDProfContext *ctx) {
         ddprof_exporter_new(ctx->worker_ctx.user_tags, ctx->worker_ctx.exp[0]));
     DDRES_CHECK_FWD(
         ddprof_exporter_new(ctx->worker_ctx.user_tags, ctx->worker_ctx.exp[1]));
-    DDRES_CHECK_FWD(pprof_create_profile(ctx->worker_ctx.pprof[0],
-                                         ctx->watchers, ctx->num_watchers));
-    DDRES_CHECK_FWD(pprof_create_profile(ctx->worker_ctx.pprof[1],
-                                         ctx->watchers, ctx->num_watchers));
+
+    // The export profiles non-optionally have the default SampleType of
```

Comment:
This comment might not match the actual implementation ;)

Reply:
Oopsies :)